EC_process/timeline.md

```diff
@@ -0,0 +1,54 @@
+# Executive Council Yearly Timeline
+
+Additional tasks that don't fit into the monthly timeline below include:
+* in-person (face-to-face) meeting
+* reassessing the strategic plan
+* annual finance audit
+
+## January
+* Review previous year’s minutes. Make sure all are approved and posted. 
+* Overlap meeting of old/new board
+* Logistics of offboarding outgoing members
+* Logistics of onboarding new members
+
+## February
+* New EC officially takes over
+* Reassess meeting procedures, including methods for setting the agenda and taking minutes
+* Previous year’s financial report for new EC
+* Select new officers
```

Comment:
This year, we waited until the March meeting, but I was going to suggest to do this in February from now on. So +1.